### ISSUE TITLE
[Enhance] Matter Network

### ISSUE THREAD

**Issue by LatentDesire** [DarkholmeTenk/TardisModIssues#534]
I forget if I had already suggested it but I didn't see it when I looked back, but it would be great if you could connect Matter Pipes and Matter Network Pipes (They function differently from each other) from Matter Overdrive through the TARDIS nodes.

I'm not sure but I think making the node function like a router from that mod would work for the matter network pipes part.

**Comment by cloakable** [comment]
Matter Plasma is a fluid, so you can already route it through the TARDIS using the Fluiduct interface; but +2 to the matter network function.

**Comment by LatentDesire** [comment]
I didn't know that about matter plasma. I was wondering how else I might be able to stockpile a large quantity of it. Would Matter Overdrive machines be able to auto detect that the fluid is in a tank from another mod, or would you need to pipe it back into a decomposer or other machine? Do the matter pipes transport other fluids as well?

**Comment by cloakable** [comment]
You would need to pipe it into the machine using the matter plasma; the machines don't really care where it's stored.
Go wild, store it in a bedrockium drum!
I use enderio ender fluid conduit for it myself.
No idea about the matter pipes, I've never actually used them.

**Comment by FoxPotato** [comment]
as long as matter overdrive has forge tanks compatibility it'll be fine :P

**Comment by cloakable** [comment]
Yeah, I have no problems storing the plasma in the tardis internal tanks, but being able to route the network through the tardis too would be good :D
